exercises/series/src/test/java/SeriesTest.java

```diff
@@ -0,0 +1,144 @@
+
+import java.util.Arrays;
+import java.util.List;
+import junit.framework.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+public class SeriesTest {
+
+    @Test
+    public void hasDigitsShort() {
+        Series sut = new Series("01234");
+        List<Integer> expected = Arrays.asList(0, 1, 2, 3, 4);
+        List<Integer> actual = sut.getDigits();
+        assertNotNull(actual);
```

Comment:
This is a really nice touch: checking for null, then non-empty, then actual contents.  Test systems like this give great signal.  Thank you, @javaeeeee!